packages/bpk-component-nudger/stories.js

```diff
@@ -59,6 +61,68 @@ class NudgerContainer extends Component {
   }
 }
 
+const options = ['economy', 'premium', 'business', 'first'];
+const compareValues = (a, b): number => {
```

Comment:
Why haven't we typed the arguments in these four functions? Only the return value is typed

Reply:
Good catch!